exporter/lightstepexporter/go.mod

```diff
@@ -5,10 +5,10 @@ go 1.14
 require (
 	github.com/census-instrumentation/opencensus-proto v0.2.1
 	github.com/golang/protobuf v1.3.5
-	github.com/lightstep/opentelemetry-exporter-go v0.1.5
+	github.com/lightstep/opentelemetry-exporter-go v0.6.2
 	github.com/stretchr/testify v1.5.1
-	go.opentelemetry.io/collector v0.3.1-0.20200522130256-daf2fc71ac65
-	go.opentelemetry.io/otel v0.2.3
+	go.opentelemetry.io/collector v0.3.1-0.20200518164231-3729dac06f74
```

Comment:
Why is this going to an older version?

Reply:
When the build is broken, Go does very odd things in the .mod files, and then after you fix the build, you're left with these artifacts. Fixed.